.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -281,7 +281,11 @@ tasks:
       - func: check
 
   - name: test
-    tags: ['required-for-publish', 'run-on-pr']
+    tags: 
+      - required-for-publish
+      - run-on-pr
+      - assigned_to_jira_team_compass_compass
```

Comment:
The tag format is `assigned_to_jira_team_<jira_project_id>_<team_tag>`. The first compass maps to jira project and the second on maps to the team name in [mothra config](https://github.com/10gen/mothra/blob/main/mothra/teams/et.yaml#L48)